projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/__mocks__/index.ts

```diff
@@ -145,5 +146,23 @@ export const caseFieldMockData: CaseField = Object.assign(new CaseField(), {
       collection_field_type: null
     })
   }),
-  display_context_parameter: '#COLLECTION(allowInsert,allowUpdate)'
+  display_context_parameter: '#COLLECTION(allowInsert,allowUpdate)',
+  value: [
+    {
+      id: '725bf4a7-9e4c-4132-b5c1-f8028cb83459',
+      value: {
+        document_url: 'https://dm-store-aat.service.core-compute-aat.internal/documents/e5366837-b3f6-492d-acbf-548730625e8f',
+        document_filename: 'Screenshot 2023-06-01 at 16.07.06.png',
+        document_binary_url: 'https://dm-store-aat.service.core-compute-aat.internal/documents/e5366837-b3f6-492d-acbf-548730625e8f/binary'
+      } as CaseFileViewDocument
```

Comment:
@DavidJayakumar - quick question... how come `CaseFileViewDocument` is involved with Query Management? Is it just that it's a convenient domain model to use?